index.php

```diff
@@ -615,123 +256,30 @@
             }
-
-            $page_loaded = true;
         } catch (Exception $e) {
             if($e instanceof DatabaseDeadlockException) {
                 error_log("DEADLOCK - retry did not solve");
                 $system->db->rollbackTransaction();
                 $system->message("Database deadlock, please reload your page and tell Lsm to fix!");
                 $system->printMessage(true);
             }
-            else if(strlen($e->getMessage()) > 1) {
+            elseif(strlen($e->getMessage()) > 1) {
                 $system->db->rollbackTransaction();
                 $system->message($e->getMessage());
                 $system->printMessage(true);
             }
         }
     }
-    else if (isset($_GET['home'])) {
-        $initial_home_view = "default";
-        if (isset($_GET['view'])) {
-            switch ($_GET['view']) {
-                case "news":
-                    $initial_home_view = "news";
-                    break;
-                case "contact":
-                    $initial_home_view = "contact";
-                    break;
-                case "rules":
-                    $initial_home_view = "rules";
-                    break;
-                case "terms":
-                    $initial_home_view = "terms";
-                    break;
-            }
-        }
-        $layout->renderBeforeContentHTML(
-            $system,
-            $player ?? null,
-            "Home",
-            render_content: false,
-            render_header: true,
-            render_sidebar: false,
-            render_topbar: false
-        );
-
-        try {
-            require('./templates/home.php');
-        } catch (RuntimeException $e) {
-            $system->db->rollbackTransaction();
-            $system->message($e->getMessage());
-            if (!$system->layout->usesV2Interface()) {
-                $system->printMessage(true);
-            }
-        }
-
-        $layout->renderAfterContentHTML($system, $player ?? null, render_content: false, render_footer: false, render_hotbar: false);
-        $page_load_time = round(microtime(true) - $PAGE_LOAD_START, 3);
-        $system->db->commitTransaction();
-    }
-    else {
-        $layout->renderBeforeContentHTML(
-            system: $system,
-            player: $player,
-            page_title: "Profile"
-        );
 
-        $system->printMessage();
-
-        try {
-            require("pages/profile.php");
-            userProfile();
-        } catch (RuntimeException $e) {
-            $system->db->rollbackTransaction();
-            $system->message($e->getMessage());
-            $system->printMessage(true);
-        }
-    }
+    // Render after content
+    $PAGE_LOAD_TIME = microtime(as_float: true) - $PAGE_LOAD_START;
+    $system->layout->renderAfterContentHTML(
+        system: $system, player: $player ?? null,
+        page_load_time: $PAGE_LOAD_TIME,
+        render_content: $RENDER_CONTENT,
+        render_hotbar: false
```

Comment:
`render_hotbar` is not a parameter on this function AFAIK

Reply:
I believe this is tied to the built-in rage bar. Looks like we can drop it because it will only render on dev.